AnnotatoBackend/Sources/App/Models/Extensions/AnnotationEntity+CRUD.swift

```diff
@@ -36,6 +36,16 @@ extension AnnotationEntity {
             }
         }
 
+        let annotationHandwritings = annotation.parts.compactMap({ $0 as? AnnotationHandwriting })
```

Comment:
Hong yao we don't need to edit the customDelete for annotation because we already have on delete cascase is that correct?

Reply:
We'll still need it as on delete cascade only works for hard deletions. To soft delete the relations properly, we will need to call custom delete on the children too.

Thanks for letting me know, I'll need to edit the custom delete for annotation